package.json

```diff
@@ -2,7 +2,9 @@
   "name": "react-onclickoutside",
   "version": "5.11.1",
   "description": "An onClickOutside wrapper for React components",
-  "main": "index.js",
+  "main": "./lib/index.js",
```

Comment:
Always test your convictions against the real world: the "bloat" you talk about is 263 bytes. That is *not* going to impact anyone, while allowing this library to ditch 10+ dependencies that this PR adds for creating the separate files.